index.bs

```diff
@@ -4341,6 +4365,12 @@ lists and names some [=authenticator types=] of particular interest.
                 <td> [=client-side credential storage modality|Client-side storage=] </td>
                 <td> [=Multi-factor capable=] </td>
             </tr>
+            <tr>
+                <th> <dfn>Passkey platform authenticator</dfn> </th>
+                <td> [=platform attachment|platform=] ({{AuthenticatorTransport|transport}} = {{AuthenticatorTransport/internal}}) or [=cross-platform attachment|cross-platform=] ({{AuthenticatorTransport|transport}} = {{AuthenticatorTransport/hybrid}})</td>
+                <td> [=client-side credential storage modality|Client-side storage=] </td>
+                <td> [=Multi-factor capable=] </td>
+            </tr>
         </tbody>
     </table>
     <figcaption>
```

Comment:
The prose below also needs to be updated - in particular the "The four combinations not named in [the table]" list, where the first item needs to be removed or updated (and the "four" needs to change too - we should probably just delete it and just say "The combinations not named [...]". I don't think it's ever been 4, actually.)

Reply:
addressed in [8813987](https://github.com/w3c/webauthn/pull/1901/commits/88139876d0418563ac8492f1eada3fc9eed4c9af)